code-of-conduct.md

```diff
@@ -0,0 +1,92 @@
+# Code of Conduct
+
+## Our Pledge
```

Comment:
Maybe just link to the existing code of conduct?
https://github.com/kubeflow/community/blob/master/CODE_OF_CONDUCT.md

Reply:
done